packages/sources/src/addSourceWizard/FinalWizard.js

```diff
@@ -34,22 +33,18 @@ const FinalWizard = ({
                     successfulMessage={ successfulMessage }
                     hideSourcesButton={ hideSourcesButton }
                     returnButtonTitle={ returnButtonTitle }
-                    progressStep={progressStep}
-                    progressTexts={progressTexts}
+                    secondaryActions={<Button variant="link" onClick={ reset }><FormattedMessage id="wizard.addAnotherSource" defaultMessage="Add another source"/></Button>}
```

Comment:
I hope that you are excited to hear that the imperative `formatMessage` from `useIntl` has better performance than any of  the `Formatted` components :wink:

Reply:
But using elements is cleaner and much faster to write. So I don't worry too much about it. :smile: